### PR TITLE
Feature(count-to): add count-to page

### PR DESCRIPTION
**Pull request by terry-ice**
- 添加count-to 功能
- 页面没有翻译

### CODE REVIEW

File: src/views/components-demo/count-to.vue [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+export default class CountToDemo extends Vue {
+  private setStartVal = 0
+  private setEndVal = 2017
+  private setDuration = 4000
+  private setDecimals = 0
+  private setSeparator = ''
+  private setSuffix = ' rmb'
+  private setPrefix = '¥ '
+
+  get _startVal() {
```
Comment (Armour) [inline src/views/components-demo/count-to.vue]: can we use camelcase? (remove the '_' prefix)

File: src/views/components-demo/count-to.vue [hunk 2]
```diff
@@ -0,0 +1,276 @@
...
+    if (this.setStartVal) {
+      return this.setStartVal
+    } else {
+      return 0
+    }
+  }
+  get _endVal() {
```
Comment (Armour) [inline src/views/components-demo/count-to.vue]: same as above

File: mock/role/routes.ts [hunk 3]
```diff
@@ -196,6 +196,12 @@ export const asyncRoutes = [
         name: 'BackToTopDemo',
         meta: { title: 'backToTop' }
       },
+      {
```
Comment (Armour) [inline mock/role/routes.ts]: See line 182-185, remove this and change line 183 to '/count-to'

File: src/views/components-demo/count-to.vue (hunk 1 above)
Comment (terry-ice) [reply src/views/components-demo/count-to.vue]: already edited

File: mock/role/routes.ts (hunk 3 above)
Comment (terry-ice) [reply mock/role/routes.ts]: already edited

File: src/router/modules/components.ts [hunk 4]
```diff
@@ -50,12 +50,12 @@ const componentsRouter: RouteConfig = {
...
     // },
-    // {
-    //   path: 'count-to',
-    //   component: () => import('@/views/components-demo/count-to'),
-    //   name: 'CountToDemo',
-    //   meta: { title: 'Count To' }
-    // },
+    {
+      path: 'count-to',
+      component: () => import(/* webpackChunkName: "CountTo" */ '@/views/components-demo/count-to.vue'),
```
Comment (Armour) [inline src/router/modules/components.ts]:
```suggestion
      component: () => import(/* webpackChunkName: "countTo" */ '@/views/components-demo/count-to.vue'),
```

File: src/views/components-demo/count-to.vue [hunk 5]
```diff
@@ -0,0 +1,276 @@
...
+      :end-val="endVal"
+      :duration="duration"
+      :decimals="decimals"
+      :separator="separator"
+      :prefix="prefix"
+      :suffix="suffix"
+      :autoplay="false"
+      class="count"
+    />
+    <div style="margin-left: 25%;margin-top: 40px;">
```
Comment (Armour) [inline src/views/components-demo/count-to.vue]:
```suggestion
    <div style="margin-left: 25%; margin-top: 40px;">
```

File: src/views/components-demo/count-to.vue [hunk 6]
```diff
@@ -0,0 +1,276 @@
...
+      >
+        suffix:
+        <input
+          v-model="setSuffix"
+          name="suffixInput"
+        >
+      </label>
+    </div>
+    <aside>
+      &lt;count-to :start-val=&#x27;{{ _startVal }}&#x27; :end-val=&#x27;{{ _endVal }}&#x27; :duration=&#x27;{{ _duration }}&#x27;
```
Comment (Armour) [inline src/views/components-demo/count-to.vue]: Remember to change here too: _startVal etc.

Comment (Armour) [inline src/views/components-demo/count-to.vue]:
```suggestion
      &lt;count-to :start-val=&#x27;{{ startVal }}&#x27; :end-val=&#x27;{{ endVal }}&#x27; :duration=&#x27;{{ duration }}&#x27;
```

File: src/views/components-demo/count-to.vue [hunk 7]
```diff
@@ -0,0 +1,276 @@
...
+      :decimals=&#x27;{{ _decimals }}&#x27; :separator=&#x27;{{ _separator }}&#x27; :prefix=&#x27;{{ _prefix }}&#x27; :suffix=&#x27;{{ _suffix }}&#x27;
```
Comment (Armour) [inline src/views/components-demo/count-to.vue]:
```suggestion
      :decimals=&#x27;{{ decimals }}&#x27; :separator=&#x27;{{ separator }}&#x27; :prefix=&#x27;{{ prefix }}&#x27; :suffix=&#x27;{{ suffix }}&#x27;
```

**Comment by Armour** [comment]
#22

File: src/router/modules/components.ts (hunk 4 above)
Comment (terry-ice) [reply src/router/modules/components.ts]: ok

**Pull request merged**